packages/mobile/src/store/nativeServices/nativeServicesCallbacks/nativeServicesCallbacks.ts

```diff
@@ -36,6 +36,7 @@ export const deviceEvents = () => {
         (channelAddress: string) => {
           emit(publicChannels.actions.setCurrentChannel({ channelAddress }))
           emit(initActions.setCurrentScreen(ScreenNames.ChannelScreen))
+          emit(messages.actions.updateMessagesVerification())
```

Comment:
It triggers only if the user tapped the notification, right? What happens for the app woken up in some other way?